components/ui/TopMenu.tsx

```
@@ -0,0 +1,118 @@
+      <div className="flex justify-between items-center px-6 py-4 top-bar opacity-0">
+        <button
+          onClick={() => setMenuOpen(!menuOpen)}
+          className="text-leather-200 hover:text-leather-100 transition-colors p-2 flex items-center gap-2"
+          aria-label="Toggle menu"
+        >
+          <svg
+            className="w-6 h-6"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+          >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)